source.js

```diff
@@ -0,0 +1,188 @@
+export const authors = [
+    {
+      name: "William Shakespeare",
+      nationality: "English"
+    },
+    {
+      name: "Jane Austen",
+      nationality: "English"
+    },
+    {
+      name: "Leo Tolstoy",
+      nationality: "Russian"
+    }
+    
+  ];
+
+  export const categories = [
+    
+    "Frontend Development",
+    "Backend Development",
+    "Data Management",
+    "Functional Programming",
+    "Object-Oriented Programming (OOP)",
+    "Security",
+    
+];
+
+
+export const books = [
+    { 
+        name: "OOP", 
+        category: "programming", 
+        price: 30, 
+        author: "Robert C. Martin", 
+        isbn: "123", 
+        issBook: { 
+            issueStatus: false, 
+            issueDate: "" 
+        }
+    },
+    { 
+        name: "Python", 
+        category: "AI", 
+        price: 36, 
+        author: "Stuart Russell", 
+        isbn: "234", 
+        issBook: { 
+            issueStatus: true, 
+            issueDate: "2020-05-20T12:00:00"  // Hard-assigned past date and time
+        }
+    },
+    { 
+        name: "React", 
+        category: "programming", 
+        price: 28, 
+        author: "Kirupa Chinnathambi", 
+        isbn: "890", 
+        issBook: { 
+            issueStatus: false, 
+            issueDate: "" 
+        }
+    },
+    { 
+        name: "JavaScript: The Good Parts", 
+        category: "programming", 
+        price: 25, 
+        author: "Douglas Crockford", 
+        isbn: "567", 
+        issBook: { 
+            issueStatus: true, 
+            issueDate: "2020-05-21T10:30:00"  // Hard-assigned past date and time
+        }
+    },
+    { 
+        name: "Clean Code", 
+        category: "programming", 
+        price: 32, 
+        author: "Robert C. Martin", 
+        isbn: "456", 
+        issBook: { 
+            issueStatus: false, 
+            issueDate: "" 
+        }
+    },
+    { 
+        name: "Artificial Intelligence: A Modern Approach", 
+        category: "AI", 
+        price: 40, 
+        author: "Peter Norvig", 
+        isbn: "789", 
+        issBook: { 
+            issueStatus: true, 
+            issueDate: "2020-05-22T15:45:00"  // Hard-assigned past date and time
+        }
+    },
+    { 
+        name: "Database Systems: The Complete Book", 
+        category: "Database", 
+        price: 38, 
+        author: "Hector Garcia-Molina", 
+        isbn: "321", 
+        issBook: { 
+            issueStatus: false, 
+            issueDate: "" 
+        }
+    },
+    { 
+        name: "The Pragmatic Programmer", 
+        category: "programming", 
+        price: 29, 
+        author: "Andrew Hunt", 
+        isbn: "654", 
+        issBook: { 
+            issueStatus: false, 
+            issueDate: "" 
+        }
+    }
+];
+
+
+
+
+export const createInputForUpdate = (type, className, placeholder, value,required )=> {
```

Comment:
This function should take an object as a parameter.  :point_down: 
 `({type, className, placeholder,  value, required })`